src/components/Editor/Breakpoint.js

```diff
@@ -7,12 +7,12 @@ import classnames from "classnames";
 import Svg from "../shared/Svg";
 
 import { getDocument, showSourceText, toEditorLine } from "../../utils/editor";
-
-const breakpointSvg = document.createElement("div");
-ReactDOM.render(<Svg name="breakpoint" />, breakpointSvg);
+//
```

Comment:
Was a comment going to be added here? if not we can remove this

Reply:
Likely a mistake on my part @bomsy...I'll remove that.